graylog2-web-interface/src/views/components/searchbar/csvexport/CSVExportSettings.jsx

```diff
@@ -0,0 +1,107 @@
+// @flow strict
+import React from 'react';
```

Comment:
We should import this as
```suggestion
import * as React from 'react';
```
So we do not get only the default export.